src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowTypeController.cs

```diff
@@ -352,7 +373,7 @@ public async Task<IActionResult> Duplicate(WorkflowTypePropertiesViewModel viewM
 
             return RedirectToAction(nameof(Edit), new
             {
-                workflowType.Id,
```

Comment:
Keep these trailing commas, here and on line 490.

Reply:
Done